interface/PLTBinaryFileReader.h

```diff
@@ -41,6 +41,7 @@ class PLTBinaryFileReader
     void SetPlaneFiducialRegion (PLTPlane::FiducialRegion);
 
     PLTPlane::FiducialRegion fPlaneFiducialRegion;
+    std::set<int>& PixelMask(){return fPixelMask;}
```

Comment:
Please make this function return a const (probably don't want things messing around with the mask after it's created).